terraform/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "vpc-cidr" { default = "" }
```

Comment:
don't use an empty default value for that, this will force an error if someone tries to deploy the script without passing the vpc-cidr variable whereas if you default to none, it will just give an error saying that the value is wrong.

Reply:
Could have a default value, but can still be replaced, or remove the default just do something like `variable "vpc_cidr" { type = string }`